api/metrics/Covid_cdd_metric.py

```diff
@@ -8,23 +8,14 @@ class CommunityMetrics:
     def __init__(self, con):
         self.con = con
 
-    def cases_by_age_given_month(self):
+    def cases_by_age_given_week(self, date):
```

Comment:
For the timeline chart, you don't need a metric that takes in date as a parameter, you want to return all the dates.